### PR TITLE
Fix:Remove unused function

### LINKED ISSUE #528: Remove unused code in app.
**Actual Behaviour**

Unused code is present in LoginVC

**Expected Behaviour**

It should be removed .

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by baquer**
Fixes #528 

Changes: remove the unused code.

Screenshots for the change: 

No SS needed.

### CODE REVIEW

**Comment by baquer** [comment]
@chashmeetsingh , @jogendra , please review the PR.

File: Susi/Controllers/LoginController/LoginVCMethods.swift [hunk 1]
```diff
@@ -256,14 +256,6 @@ extension LoginViewController: UITextFieldDelegate {
         passwordTextField.text = ""
     }
 
-    // Show sign up controller
-    func presentSignUpController() {
-        clearFields()
-
-        UserDefaults.standard.setValue(emailTextField.text, forKey: ControllerConstants.UserDefaultsKeys.typedEmailAdress)
```
Comment (jogendra) [inline Susi/Controllers/LoginController/LoginVCMethods.swift]: I think we are using this UserDefault value to pass the email address

Comment (baquer) [reply Susi/Controllers/LoginController/LoginVCMethods.swift]: But, this function is not referred or called anywhere.

Comment (jogendra) [reply Susi/Controllers/LoginController/LoginVCMethods.swift]: Okay!

**Comment by baquer** [comment]
> Please try making PR with significant change

Okay.

**Pull request merged**